src/Cache.cpp

```diff
@@ -4045,6 +4045,15 @@ Cache::updateUserKeys(const std::string &sync_token, const mtx::responses::Query
     }
 }
 
+void
+Cache::markUserKeysOutOfDate(const std::vector<std::string> &user_ids)
+{
+    auto txn = lmdb::txn::begin(env_);
+    auto db  = getUserKeysDb(txn);
+    markUserKeysOutOfDate(txn, db, user_ids, "");
```

Comment:
I think setting the batch_token to the current sync token would make the most sense here?